### PR TITLE
Responsive sidebar

### PR DESCRIPTION
**Pull request by yuriikaterlin**
created mobile toolbar for adaptive sidebar
fixed footer with horizontal scroll
fixed responive ubs-user-messages

### CODE REVIEW

**Comment by andreikazmirovich** [comment]
Could you please add some screenshots?

File: src/app/shared/ubs-base-sidebar/ubs-base-sidebar.component.html [hunk 1]
```diff
@@ -1,4 +1,11 @@
 <div class="main">
+  <mat-toolbar class="mobile-toolbar">
+    <button mat-icon-button class="mobile-menu" aria-label="Sidebar menu" (click)="toggleSideBar()">
+      <mat-icon class="mobile-icon">{{ !drawer.opened ? 'menu' : 'close' }}</mat-icon>
+    </button>
+    <span class="example-spacer"></span>
```
Comment (andreikazmirovich) [inline src/app/shared/ubs-base-sidebar/ubs-base-sidebar.component.html]: Do you need it? If yes, I believe the class should be renamed not to be `example-...`

File: src/app/shared/ubs-base-sidebar/ubs-base-sidebar.component.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { AfterViewInit, Component, ElementRef, Input, OnDestroy, ViewChild } from '@angular/core';
+import { AfterViewInit, Component, ElementRef, HostBinding, Input, OnDestroy, ViewChild } from '@angular/core';
```
Comment (andreikazmirovich) [inline src/app/shared/ubs-base-sidebar/ubs-base-sidebar.component.ts]: I believe you don't need to import this `HostBinding`

File: src/app/ubs-user/ubs-user-messages/ubs-user-messages.component.scss [hunk 3]
```diff
@@ -237,3 +237,47 @@
...
+    justify-content: space-between;
+    margin-top: -32px;
+
+    div {
+      color: var(--ubs-primary-grey);
+      font-size: 14px;
+      font-style: normal;
+      font-weight: 700;
+      line-height: 20px;
+      letter-spacing: 0.10000000149011612px;
```
Comment (andreikazmirovich) [inline src/app/ubs-user/ubs-user-messages/ubs-user-messages.component.scss]: What a strange number
You should change it to something normal

**Review by andreikazmirovich — APPROVED** [review]

File: src/app/ubs-user/ubs-user-messages/ubs-user-messages.component.scss (hunk 3 above)
Comment (yuriikaterlin) [reply src/app/ubs-user/ubs-user-messages/ubs-user-messages.component.scss]: Done

File: src/app/shared/ubs-base-sidebar/ubs-base-sidebar.component.ts (hunk 2 above)
Comment (yuriikaterlin) [reply src/app/shared/ubs-base-sidebar/ubs-base-sidebar.component.ts]: Done

**Comment by yuriikaterlin** [comment]
> Could you please add some screenshots?

Please see the attached screenshot
sidebar

**Review by NewArtMask — APPROVED** [review]

**Review by andreikazmirovich — APPROVED** [review]
Good job, looks great!

**Pull request merged**